Resources/Locale/en-US/deltav/prototypes/access/accesses.ftl

```diff
@@ -5,3 +5,4 @@ id-card-access-level-prosecutor = Prosecutor
 id-card-access-level-clerk = Clerk
 id-card-access-level-justice = Justice
 id-card-access-level-corpsman = Corpsman
+id-card-access-level-pharmacist = Pharmacist
```

Comment:
Similar to above, dropping this and renaming the existing Psychology access to something like "Psychiatry".